node-operators/parachain-node/parachain-node.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Parachain Node
+description: How to run a Parachain node for the Moonbeam Network
+---
+
+# Run a Parachain Node on Moonbeam
+
+(cool image) 
+
+## Introduction
+
+With the v5 release of Moonbeam, we now support multiple collators and rpc nodes.  We invite everyone to spin up a node and join the Moonbase network.  Collation slots will be limited for now, but anyone can run a full node to connect to their own private RPC endpoint.  PureStake will be hosting the relay chain in PoA mode, and the instructions below explain how to run our parachain and connect it to the Moonbase network.  As development progresses, Kusama and then Polkadot will become the relay chains.  Here's how we will name these upcoming environments: 
```

Comment:
"rpc nodes" is a term I've only heard here. I think "full node" is more common. Just my 2c.